src/lightning/pytorch/loggers/neptune.py

```diff
@@ -468,7 +468,13 @@ def finalize(self, status: str) -> None:
             # initialized there
             return
         if status:
-            self.run[self._construct_path_with_prefix("status")] = status
+            if _NEPTUNE_AVAILABLE:
+                from neptune.exceptions import InactiveRunException
+            else:
+                from neptune.new.exceptions import InactiveRunException
```

Comment:
Shouldn't this be
```suggestion
            elif _NEPTUNE_CLIENT_AVAILABLE:
                from neptune.new.exceptions import InactiveRunException
```
because if neptune is not  available, you wouldn't be able to import the package

Reply:
Yeah, thanks for raising this. Let me wait for: https://github.com/Lightning-AI/pytorch-lightning/pull/19131 as this may require some adjustments after merging.

Reply:
lets roll this on its own and do not chain it with other PR :)